source/developers/references/developer_guide/process/code-considerations.rst

```diff
@@ -9,15 +9,91 @@ as they are building new, or modifying existing, functionality.
    :local:
    :depth: 1
 
+==================
+Basic Code Quality
+==================
+
+As described in :doc:`overview`, your code should meet basic Open edX project
+standards: it should be accessible, internationalized, and meet the concerns of
+analytics.
+
+* `Internationalization Coding Guidelines`_
+* `RTL UI Best Practices`_
+* `Accessibility Guidelines`_
+* `Analytics Guidelines`_
+* `Eventing Design and Review Process`_
+
+.. _Contributing to the Documentation for your Open Source Feature:
+
+=====================
+Feature Documentation
+=====================
+
+Documentation can occur in multiple places - in code, in decision records, or in
+formal feature documentation.
+
+`OEP-19 Developer Documentation`_ describes the various ways to provide
+documentation of your code and features to a developer audience. This includes
+API documentation, decision records (ADRs and OEPs), and README files. Of
+course, you should always provide detailed, informative comments within your
+code and excellent docstrings for your functions and classes!
+
+If you have developed a new feature, created an XBlock, or otherwise see that a
+topic is missing in this documentation, you can create one or more new pages in
+this document and submit a pull request to have the content added. Follow the
+guidelines in :doc:`../../../documentors/references/doc_guidelines`.
+
+**TODO** How do I add a note about my new feature for the next named release release notes?
```

Comment:
@pdpinch I know you're not writing release notes this time around, but I'm not sure who else to ask -- if I develop a new feature, is there a good way of letting the release note writer(s) know about it?

Reply:
Under the [Release Planning](https://openedx.atlassian.net/wiki/spaces/COMM/pages/13205845/Open+edX+Release+Planning) wiki page, there is usually a page for the next release, for example the [Redwood Wiki Page](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3890380898/Next+Release+Redwood).  This is the place to put any changes related to deprecations, operational notes or new features that you want to highlight.  The release notes person usually looks at this as a part of the process for generating notes.

Reply:
Yeah, oddly I already added that later in this page.